test/ctia/test_helpers/core.clj

```diff
@@ -511,3 +512,17 @@
       (f)
       (reset! uuid-counter
               uuid-counter-start))))
+
+(defn naive-longest-common-suffix
```

Comment:
this function seems only used in `core_test`, I would move it in this `ns`

Reply:
Done.